src/ShiftedProximalOperators.jl

```diff
@@ -56,6 +63,16 @@ function (ψ::ShiftedProximableFunction)(y)
   return ψ.h(ψ.xsy)
 end
 
+function (ψ::ShiftedCompositeProximableFunction)(y)
+  return ψ.h(ψ.b + ψ.A * y)
+end
+
+function (ψ::CompositeProximableFunction)(y)
+  z = similar(ψ.b)
```

Comment:
Preallocate.

Reply:
If I add z as an argument, there will be problems later with R2 and overall most solvers compute $\psi(y)$ at some point without calling an extra argument.